Filter/DateTimeRangeFilter.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
```

Comment:
No your name can be put on top of the class keyword: like this https://github.com/sonata-project/SonataAdminBundle/blob/master/Admin/Admin.php#L53

We do not ask people to leave there names in the headers files, but if they do, we keep there names. contributors can be seen here https://sonata-project.org/contributors or on the github pages.